src/strategies/index.ts

```diff
@@ -405,7 +406,8 @@ const strategies = {
   juicebox,
   'snet-farmers': snetFarmers,
   'snet-stakers': snetStakers,
-  "snet-liquidity-providers": snetLiquidityProviders
+  'snet-liquidity-providers': snetLiquidityProviders,
```

Comment:
where is `snet-liquidity-providers` strategy? would be great if you can open a seperate PR for that :)

Reply:
sir I just added our strategy to the bottom of the list :)